### DISCUSSION TITLE
Deleting API Tokens yields an error

### DISCUSSION THREAD

**Discussion by divemasterza** [kimai/kimai#4811] [General]
### Describe the issue

Deleting a token generates a forbidden modal - (nothing shows in the log)

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).
- [X] I've searched for similar issues in this repository.

### Kimai version

2.15.0

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.2

### Logfile

nothing logged

### Screenshots

![CleanShot 2024-04-22 at 07 25 14](https://github.com/kimai/kimai/assets/5737056/e0aaaffb-b1da-493f-99dd-affef6c285d7)

**Comment by kevinpapst** [comment]
This feature is brand new, I'd be weird if I missed that.
And I tested in the demo and it doesn't. Neither there nor in my own cloud or in my dev environment.

**Comment by divemasterza** [comment]
So some more clarifications: 

- I can create a token for myself, and I can delete it.
- I can create a token for a user, but cannot delete it.

**Reply by kevinpapst** [reply]
Thanks for the clarification 👍 

> I can create a token for a user, but cannot delete it.

That was the missing piece of the puzzle, now I can see it as well.

Will be fixed with the next release.
